libraries/Servo/src/Servo.h

```diff
@@ -110,8 +110,9 @@ class Servo {
     int read();                        // returns current pulse width as an angle between 0 and 180 degrees
     int readMicroseconds();            // returns current pulse width in microseconds for this servo (was read_us() in first release)
     bool attached();                   // return true if this servo is attached, otherwise false
-  private:
+  protected:
```

Comment:
I don't see a reason to change this attribute to protected instead of private.
This variable should not be modified outside of the class.
I can understand that you need to read this variable, but in this case we can rather implement an accessor.

Reply:
A getter function would be fine too. I'm not a professional c++ programmer, so I thought a protected var would be fine.